src/Faker/Provider/es_PE/Company.php

```diff
@@ -65,4 +65,22 @@ public function bs()
 
         return implode(' ', $result);
     }
+
+    /**
+     * Generate a REG. UNICO DE CONTRIBUYENTES (RUC) number
+     *
+     * Doesn't include a checksum, as peruvians commonly use only the first
+     * 8 digits.
+     *
+     * @example '10164090581'
+     * @example '20105895410'
+     *
+     * @see http://www2.sunat.gob.pe/pdt/pdtModulos/independientes/p695/TipoDoc.htm
+     */
+    public static function ruc($personaJuridica = true)
```

Comment:
Can be typed as bool since we support 7.1+ only.

Reply:
Done